app/src/main/res/layout/item_left_conversation.xml

```diff
@@ -22,16 +24,17 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginBottom="@dimen/dp_8"
-        android:layout_marginEnd="@dimen/dp_56"
-        android:layout_marginStart="@dimen/dp_8">
+        android:layout_marginEnd="@dimen/dp_56">
 
         <ImageView
             android:id="@+id/image_recording_audio"
             android:layout_width="@dimen/dp_24"
             android:layout_height="@dimen/dp_24"
-            android:background="@drawable/ic_audio_conversation"
+            android:background="@drawable/bg_menu_media_creating_conversation"
             android:contentDescription="@string/description_recording_audio_convention"
-            android:onClick="@{() -> viewModel.onAudioConversationClick(conversations)}"
+            android:onClick="@{(v) -> viewModel.showMenu(v, position, conversations)}"
+            android:padding="@dimen/dp_4"
+            android:src="@drawable/ic_menu_media"
```

Comment:
ERROR: [VectorDrawableCompat] When using VectorDrawableCompat, you need to use `app:srcCompat`.
 To use VectorDrawableCompat, you need to make two modifications to your project. First, set `android.defaultConfig.vectorDrawables.useSupportLibrary = true` in your `build.gradle` file, and second, use `app:srcCompat` instead of `android:src` to refer to vector drawables.